### DISCUSSION TITLE
"You don't have access to GitHub Copilot" But I purchased Github Copilot for one year already!

### DISCUSSION THREAD

**Discussion by jeffreyouni** [community/community#112445] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I purchased github copilot one year service, but suddenly all my github copilot saying that I don't have token or access to github copilot!

This is the error in vscode:
"2024-03-13 15:27:19.561 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-13 15:27:19.563 [error] [ghostText] provideInlineCompletionItems: CopilotAuthError: Contact Support. You are currently logged in as jeffreyouni."

This is the error in intellj IDEA:
![image](https://github.com/community/community/assets/35049781/9146f0aa-366c-425c-8665-7823117f950d)

This is the add-on billing:
![image](https://github.com/community/community/assets/35049781/a1515888-3482-48be-b766-3f7cb205e86e)

This is Copilot page:
![image](https://github.com/community/community/assets/35049781/d8a559ae-65b0-43ff-b374-1971bff51b2a)

I didn't change anything, it was working for a week but suddenly stopped.

PS: I couldn't find a the customer service to help so I come here.

Thank you for your support.

**Comment by Ryanyamg** [comment]
me  too

**Comment by yozurukt** [comment]
me too
![2d53136affe221dfe75c3393aba212bb](https://github.com/community/community/assets/59713727/ce55180d-d3a4-456d-b8ed-379392ce1b3e)
![637fec5f8054ee018af3db18cfb50a7e](https://github.com/community/community/assets/59713727/d0edb7d0-bca5-43ce-9ae0-aab978f1e84f)

**Comment by davecheney** [comment, marked as answer]
I'm sorry but we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact

**Comment by OnHaiping** [comment]
我也是，谁来帮帮我?
monry is not free~

**Reply by John15380** [reply]
怎么提交工单啊
